.scripts/phpcs.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+# we need this script to fail if any command fails
+# when a command fails the variable `err` will turn 1
+err=0
+trap 'err=1' ERR
+
+# ensure that files do not exist
+# -f makes sure it returns 0 if the files didn't exist
+rm -f diff.txt phpcs.json
+
+# get diff from last travis build
+# if PR, get diff from the base branch
+git diff $TRAVIS_COMMIT_RANGE > diff.txt
+
+# get all the style errors
+# `|| true` makes sure it returns 0 even when phpcs fails
+./vendor/bin/phpcs ./ --report=json --report-file=phpcs.json || true
```

Comment:
here you should use `TRAVIS_BUILD_DIR` to because if this script is not executed on root bash causes a error.